matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/model/OlmInboundGroupSessionWrapper.kt

```diff
@@ -138,14 +146,11 @@ internal class OlmInboundGroupSessionWrapper : Serializable {
      * @return the exported data
      */
     fun exportSession(messageIndex: Long): String? {
```

Comment:
I wonder if it's worth using the `orNull` convention the standard lib uses for this case `exportSessionOrNull()`

what do you think?